odoo/addons/base/models/res_partner.py

```diff
@@ -275,8 +275,7 @@ def _avatar_get_placeholder_path(self):
 
     @api.depends('is_company', 'name', 'parent_id.display_name', 'type', 'company_name')
     def _compute_display_name(self):
-        diff = dict(show_address=None, show_address_only=None, show_email=None, html_format=None, show_vat=None)
-        names = dict(self.with_context(**diff).name_get())
+        names = dict(self.with_context({}).name_get())
```

Comment:
Maybe add a comment?
```suggestion
        # retrieve name_get() without any fancy feature
        names = dict(self.with_context({}).name_get())
```

Reply:
:+1: